### PR TITLE
add get routes for survey

### PR DESCRIPTION
**Pull request by patil-neha-3**

### CODE REVIEW

File: backend/routes/survey.py [hunk 1]
```diff
@@ -15,7 +15,20 @@ async def create_survey(request: Request, survey: Survey = Body(...)):
...
 
 
-@survey_router.get("/", response_description="Get all survey")
-async def get_survey(request: Request):
-    # implement this later
-    return JSONResponse(content={"hi":"Neha"})
+@survey_router.get("/", response_description="Get all surveys")
+async def list_surveys(request: Request):
+    surveys = []
+    for survey in request.app.mongodb["Survey"].find().to_list():  # would i need to add pagination?
```
Comment (DheerajKumar19) [inline backend/routes/survey.py:21]: ...why?

This would work perfectly fine, no?
```
return request.app.mongodb["Survey"].find().to_list()
```

**Review by DheerajKumar19 — CHANGES_REQUESTED** [review]
I've left a comment about a strange bit of code. Please do look at it and then let me know.

File: backend/routes/survey.py (hunk 1 above)
Comment (patil-neha-3) [reply backend/routes/survey.py:21]: if the list of surveys is too long, i'm guessing the app could crash because the response size would be too big

Comment (DheerajKumar19) [reply backend/routes/survey.py:21]: I mean we can add a parameter to just set the number of surveys to 5 or 10 per call to avoid that right?

**Review by skhoylow8 — APPROVED** [review]

**Pull request merged**